### PR TITLE
Use guard clauses to reduce nesting

### PR DESCRIPTION
**Pull request by xeome**
Use guard statements and inverted if statements to reduce nesting in code and make it easier to understand. Guard statements are statements placed at the beginning of a function that check whether certain conditions are met and return a value immediately if they are not. Inverted if statements are if statements written in such a way that the "happy path" is written first, followed by error handling.

These techniques help to make code more readable and maintainable, as they reduce the amount of indentation and make it easier to follow the flow of the code. It can be difficult to understand what is going on in a nested structure, but by using guard statements and inverted if statements we can simplify the code and make it more intuitive.

### CODE REVIEW

File: src/MainWindow.py [hunk 1]
```diff
@@ -158,34 +166,40 @@ def cmb_devices_changed(self, combobox):
...
-                        self.builder.get_object("mode_installer").set_current_page(1)
-                    self.builder.get_object("mode_label").set_text(tr("{} not found").format("grub-x86_64-amd64-efi"))
+
+        if not tree_iter:
+            return
+
+        model = combobox.get_model()
+        self.writeMode = model[tree_iter][0]  # 0:DD, 1:Iso
+        if self.writeMode == 0:
+            self.writeMode = "ImageWriter.py"
```
Comment (eminfedar) [inline src/MainWindow.py:176]: Please add `return` after this line and remove Line 182-183.

File: src/MainWindow.py [hunk 2]
```diff
@@ -158,34 +166,40 @@ def cmb_devices_changed(self, combobox):
...
+        elif self.writeMode == 1:
+            self.writeMode = "ISOCopier.py"
+        elif self.writeMode == 2:
+            self.writeMode = "WinUSB.py"
+
+        if self.writeMode == 0:
+            return
```
Comment (eminfedar) [inline src/MainWindow.py]: Remove this and add this return to first if == 0 clause.

File: src/main.py [hunk 3]
```diff
@@ -19,18 +19,23 @@ def do_activate(self):
...
-                print("File not exists : " + file.get_path())
-        else:
+        if filecount != 1:
             print("Only one file.")
+            return
+        
+        file = files[0]
+
+        if not os.path.exists(file.get_path()):
+            print("File not exists : " + file.get_path())
```
Comment (eminfedar) [inline src/main.py]: Little translation fix (not your fault)
```suggestion
            print("File does not exist: " + file.get_path())
```

**Review by eminfedar — CHANGES_REQUESTED** [review]
I think this PR is good. Guard clauses are "generally" good practice.

I just mentioned couple changes. After you fix these, this PR seems good to me.

**Comment by xeome** [comment]
Thanks for the suggestions, I have made the changes that you requested.

**Comment by eminfedar** [comment]
Thanks for your efforts @xeome.

@fthaltun it seems Ok to me.

**Comment by fthaltun** [comment]
Thanks for your contribution @xeome.
Also thanks @eminfedar.

**Pull request merged**